posthog/warehouse/api/saved_query.py

```diff
@@ -282,6 +283,9 @@ def update(self, instance: Any, validated_data: Any) -> Any:
             if activity_log:
                 self.context["activity_log"] = activity_log
 
+            if sync_frequency and sync_frequency != "never":
+                recreate_model_paths(view)
```

Comment:
Is this related to the PR?

Reply:
Yes, this is to check if we need to recreate the model paths. I left a block comment in the `recreate_model_paths` definition about this one.